chain/client/src/client.rs

```diff
@@ -848,6 +853,10 @@ impl Client {
         &mut self,
         partial_encoded_chunk: MaybeValidated<PartialEncodedChunk>,
     ) -> Result<Vec<AcceptedBlock>, Error> {
+        self.record_receive_chunk_timestamp(
```

Comment:
It's possible that for a block and a shard, a node needs to receive multiple PartialEncodedChunk before the chunk is complete. So it's the best to call `record_receive_chunk_timestamp` when the chunk is actually ready, here https://github.com/near/nearcore/blob/master/chain/client/src/client.rs#L880